src/Data/LLVM/BitCode/Bitstream.hs

```diff
@@ -90,14 +90,21 @@ data Bitstream = Bitstream
   , bsEntries  :: [Entry]
   } deriving (Show)
 
+runGet :: BG.Get a -> S.ByteString -> Either String a
```

Comment:
Could this version of `runGet` be more succinctly implemented in terms of `binary`'s [`runGetOrFail`](https://hackage.haskell.org/package/binary-0.10.0.0/docs/Data-Binary-Get.html#v:runGetOrFail)?

Reply:
`runGetOrFail` works over lazy bytestrings, this `runGet` works over strict bytestrings.

I can try and implement it by converting the bytestring, I'll see if this has a performance impact or not.

Reply:
Thanks! And sorry to give you more work to do. Then again, I imagine that `llvm-pretty-bc-parser` isn't the only `cereal`-using library that uses `runGet`, so it's likely we'll have to tackle this elsewhere. If we do, having some performance data on this would be helpful.